src/main.js

```diff
@@ -0,0 +1 @@
+console.log("started git-explore");
```

Comment:
```suggestion
console.log("started git-explore");

```

Newlines at end of file are good. Really just doing this to test making a suggestion though.